config/development.js

```diff
@@ -36,9 +37,24 @@ module.exports = {
     watchDirectory: path.resolve(__dirname, '..', 'server'),
   },
 
+  aws: {
+    credentials: {
+      region: 'eu-west-1',
+      accessKeyId: 'test',
+      secretAccessKey: 'dev',
+    },
+    s3: {
+      params: {
+        Bucket: 'localhost',
```

Comment:
It's slightly misleading to call this localhost, better off calling it `test-bucket`